.github/workflows/pr-textlint.yml

```
@@ -62,24 +62,19 @@ jobs:
             }
             console.log("call issues.listComments:")
             console.log(issues_listComments_params)
-            github.issues.listComments(issues_listComments_params).then(response =>
-              response.data.filter(
-                comment=>comment.user.id==41898282 && comment.body.startsWith('日本語の')
-              ).map(
-                comment=>comment.id
-              ).forEach(
-                comment_id => {
-                  const issues_deleteComment_params = {
-                    comment_id: comment_id,
-                    owner: context.repo.owner,
-                    repo: context.repo.repo
-                  }
-                  console.log("call issues.deleteComment:")
-                  console.log(issues_deleteComment_params)
-                  github.issues.deleteComment(issues_deleteComment_params)
-                }
-              )
-            )
+            const response = await github.issues.listComments(issues_listComments_params)
+            await Promise.all(response.data.filter(
+              comment=>comment.user.id==41898282 && comment.body.startsWith('日本語の')
+            ).map(async comment => {
+              const issues_deleteComment_params = {
+                comment_id: comment.id,
+                owner: context.repo.owner,
+                repo: context.repo.repo
+              }
+              console.log("call issues.deleteComment:")
+              console.log(issues_deleteComment_params)
```

今回の PR とは関係ないけど `console.log` は複数の引数を取れるので1行にまとめたい気持ちがある

```suggestion
              console.log("call issues.deleteComment:", issues_deleteComment_params)
```

---

この書き方は知らなかった。
他のところにも導入します。

---

まとめました: https://github.com/dev-hato/hato-bot/pull/696/commits/1ae56b5aaa6bbac03d1b13d345a4e678c983514a